packages/next/src/build/entries.ts

```diff
@@ -12,7 +12,7 @@ import type {
 import type { LoadedEnvFiles } from '@next/env'
 import type { AppLoaderOptions } from './webpack/loaders/next-app-loader'
 
-import chalk from 'next/dist/compiled/chalk'
```

Comment:
@styfle IMHO we don't need destructuring import here. The dependency after being ncc-ed is not tree-shakable anyway.

Reply:
@SukkaW Is there any harm in destructuring here? 

IMO, it looks better to do `yellow(bold('Warning: '))` instead of `pc.yellow(pc.bold('Warning: '))`

Reply:
> @SukkaW Is there any harm in destructuring here?
> 
> IMO, it looks better to do `yellow(bold('Warning: '))` instead of `pc.yellow(pc.bold('Warning: '))`

I personally prefer the latter syntax, just like previously how chalk is used. But it does no harm anyway.

Reply:
I just pushed a change in [5df0f9e](https://github.com/vercel/next.js/pull/55992/commits/5df0f9eadc47af8f619af5c9483df5dca74d89bb) which should make it tree-shakable now.